### PR TITLE
Code tweaks

### PR DESCRIPTION
**Pull request by dkackman**
Various changes to improve efficiency and clarity of code. Add additional error handling.

### CODE REVIEW

**Comment by L4bb3rs** [comment]
Various changes to simplify code and improve speed.

File: swarm/output_processor.py [hunk 1]
```diff
@@ -56,26 +58,27 @@ def get_results(self):
...
+                thumbnail, "image/jpeg", "web_low")
             results["inference_video"] = make_result(
                 video_buffer, thumbnail_buffer, "video/webm"
             )
 
         return results
 
 
-def make_result(buffer, thumb, content_type):
+def make_result(_buffer, thumb, content_type):
```
Comment (dkackman) [inline swarm/output_processor.py:70]: why _buffer?

File: swarm/initialize.py [hunk 2]
```diff
@@ -94,9 +118,5 @@ def get_models_from_hive(hive_uri):
 
         return data["language_models"] + data["models"]
     except Exception as e:
-        print(e)
-        logging.error(e)
+        print(f"Failed to fetch known model list from {hive_uri}: {e}")
         return []
-
-
-asyncio.run(init())
```
Comment (dkackman) [inline swarm/initialize.py:102]: without this how does `init` get invoked? (This file runs as a module `python-m swarm.intialize`) Perhaps we add a `if __name__ == "__main__":` for correctness sake but this code isn't (or shouldn't be imported anywhere.

**Review by dkackman — COMMENTED** [review]
add a couple of questions

File: swarm/output_processor.py (hunk 1 above)
Comment (L4bb3rs) [reply swarm/output_processor.py:70]: Changed due to usage in BytesIO although I have reviewed again and I don't think its a required change so its your choice. It was amended throughout the script.

File: swarm/initialize.py (hunk 2 above)
Comment (L4bb3rs) [reply swarm/initialize.py:102]: Error when reviewing code.

**Pull request merged**